src/pages/admin-ui-sdk/installation.md

```diff
@@ -1,6 +1,92 @@
 ---
-title: Installation
-description: 
+title: Install Adobe Commerce Admin UI SDK
+description: Learn how to install the Adobe Commerce Admin UI SDK.
 ---
 
-# Installation
+# Install Adobe Commerce Admin UI SDK
+
+This topic describes the basic steps you need to install the Admin UI SDK.
+
+## Prerequisites
+
+### Adobe Commerce
+
+* Adobe Commerce on cloud infrastructure or on premises: 2.4.5+
```

Comment:
Did we double check that the AdminIMS module is available for 2.4.5?

Reply:
IMS support is mentioned in the 2.4.5 release notes

https://experienceleague.adobe.com/docs/commerce-operations/release/notes/adobe-commerce/2-4-5.html?lang=en#adobe-commerce-integration-with-adobe-ims